superdesk/vocabularies/vocabularies.py

```diff
@@ -115,14 +124,24 @@ class VocabulariesService(BaseService):
 
     system_keys = set(DEFAULT_SCHEMA.keys()).union(set(DEFAULT_EDITOR.keys()))
 
+    def _validate_items(self, update):
+        if 'schema' in update and 'items' in update and \
+                ('name' in update['schema'] or 'qcode' in update['schema']):
```

Comment:
what about setting what fields are required in the schema like:
```
{'name': {'required': true}, 'qcode': {'required': true}}
```
instead of hardcoding there validation for only name and qcode. so we could use it also for other vocabularies which don't use name/qcode but have required fields